terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
reading new line
read line :3 a 1 b 2
defcount :3

Symbol : aoffset : 2
	val :1

Symbol : boffset : 6
	val :2
reading new line
read line :

Symbol : 